app/services/course_exports/prepare_students_export_service.rb

```diff
@@ -150,6 +150,14 @@ def student_report_link(student)
       "oooc:=HYPERLINK(\"#{report_path(student)}\"; \"#{student.id}\")"
     end
 
+    def latest_user_standing(user)
+      user.user_standings.where(archived_at: nil).last
+    end
+
+    def school_default_standing(user)
+      Standing.find_by(school: user.school, default: true)
+    end
```

Comment:
This can be cached as the export can only have students from one school

Reply:
I assume caching means here memoization 

I think it will not work if we cache because it will use the same result even when the next student has a standing log. 🤔

Will test.

Reply:
Yes, it can be cached https://github.com/pupilfirst/pupilfirst/pull/1468/commits/568d31a178521bee41d2b5121d2c19c944abf748